src/components/SummaryTable.js

```diff
@@ -28,6 +28,10 @@ const StyledTableRow = styled("div")({
   color: `${COLOUR.GREY_1}`
 });
 
+const StyledTableCellChange = styled(Table.Cell)({
```

Comment:
Not too important (so up to you whether you change it), but for future reference I think the Emotion team recommend using the backtick syntax instead of brackets and object syntax, because it allows you to write pure CSS rather than the weird camelCase alternatives:

```
const StyledTableCellChange = styled(Table.Cell)`
    min-width: 70px;
`
```